### PR TITLE
Add Filtering to Users on PolicyDetails

### PR DESCRIPTION
**Pull request by adfost**
Screen Shot 2021-06-22 at 6 02 40 PM

Add filtering to list users with policy.

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/TenantDetails/PodDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,351 @@
+// This file is part of MinIO Console Server
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/PodDetails.tsx:1]: please remove this file and send only the changes if any, this was moved, rebase to master to find out the new location

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Policies/PolicyDetails.tsx [hunk 2]
```diff
@@ -191,14 +199,22 @@ const PolicyDetails = ({
...
-        });
+      if (loadingUsers) {
+        api
+          .invoke("GET", `/api/v1/policies/${policyName}/users`)
+          .then((result: any) => {
+            var resultUserArray = [];
+            for (var i = 0; i < result.length; i++) {
+              var added: UserName = { name: result[i] };
+              resultUserArray.push(added);
+            }
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/PolicyDetails.tsx:210]: Please use a`.map` instead

File: portal-ui/src/screens/Console/Users/types.tsx [hunk 3]
```diff
@@ -25,6 +25,9 @@ export interface User {
   policy?: string[];
 }
 
+export interface UserName {
+  name: string;
+}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/types.tsx:30]: There is no need to create an object to make filtering, you can apply it in any array of strings (as the one that users list returns)

**Review by bexsoft — CHANGES_REQUESTED** [review]
Please add more information to this PR as it doesn't refer a task or tells us what is doing

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/RewindEnable.tsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
+// This file is part of MinIO Console Server
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/RewindEnable.tsx:1]: why are you adding this file?

**Pull request closed without merging**